src/libNode/Node.cpp

```diff
@@ -1753,14 +1784,15 @@ bool Node::ProcessTxnPacketFromLookup([[gnu::unused]] const bytes& message,
 
   // Avoid using the original message for broadcasting in case it contains
   // excess data beyond the TxnPacket
-  bytes message2 = {MessageType::NODE, NodeInstructionType::FORWARDTXNPACKET};
-  if (!Messenger::SetNodeForwardTxnBlock(
-          message2, MessageOffset::BODY, epochNumber, dsBlockNum, shardId,
-          lookupPubKey, transactions, signature)) {
-    LOG_EPOCH(WARNING, m_mediator.m_currentEpochNum,
-              "Messenger::GetNodeForwardTxnBlock failed.");
-    return false;
-  }
+  // bytes message2 = {MessageType::NODE,
```

Comment:
Good optimization (similar to issue 629). Anyway can also delete this code block now to clean up.